src/modules/account/components/AccountDetails/AccountOverview/index.js

```diff
@@ -12,7 +10,7 @@ import BoxContent from '@theme/box/content';
 import Transactions from '@transaction/components/Explorer';
 import TransactionEvents from '@transaction/components/TransactionEvents';
 import { selectActiveToken, selectSettings, selectTransactions } from 'src/redux/selectors';
-import InfoBanner from '@common/components/infoBanner/infoBanner';
+import SwippableInfoBanner from '@common/components/infoBanner/swippableInfoBanner';
 import banners from './banners';
 import 'swiper/css';
 import 'swiper/css/pagination';
```

Comment:
Swiper styles are redundant as they're already imported in SwippableInfoBanner and should therefore be removed

Reply:
Resolved